sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_anonymous_exchange_client.py

```diff
@@ -49,10 +46,10 @@ async def get_acr_access_token(self, challenge: str, **kwargs: Dict[str, Any]) -
 
     async def exchange_refresh_token_for_access_token(
         self,
-        refresh_token: str = None,
-        service: str = None,
-        scope: str = None,
-        grant_type: str = TokenGrantType.PASSWORD,
+        refresh_token: Optional[str] = None,
+        service: Optional[str] = None,
+        scope: Optional[str] = None,
+        grant_type: Optional[str] = TokenGrantType.PASSWORD,
```

Comment:
```suggestion
        grant_type: str = TokenGrantType.PASSWORD,
```

Reply:
Here changed to string type without default value.